src/uct/api/uct.h

```diff
@@ -287,30 +288,57 @@ enum uct_am_flags {
 
 
 /**
- * @ingroup UCT_AM
- * @brief Active message callback flags.
+ * @ingroup UCT_RESOURCE
+ * @brief Callback flags.
  *
- * List of flags for active message callback
+ * List of flags for a callback
```

Comment:
pls add a period at the end.

Also here is suggested re-wording for the following sentence:
  A callback must have either the SYNC or ASYNC flag set.